### PR TITLE
Update ToS and PP

### PR DESCRIPTION
**Pull request by davegson**

### CODE REVIEW

File: privacy/index.md [hunk 1]
```diff
@@ -4,150 +4,562 @@ title: Privacy Policy
...
+the data, we collect from the services we provide.
 
-We do not collect any PII (personal identifiable information). We only have what you give us willingly and on purpose. For example, when you sign up on our forum or subscribe to Gate17.
+**Website:**
 
-## How we guide the process of data collection
+The data we collect from Website Visitors:
 
-Our goal is to protect user privacy! We are careful about what we collect, and how we collect it. As a general principle, if we find a way to collect less data and still get the little information we need, we will do it.
+Includes Domains: <https://safing.io>, [https://docs.safing.io](https://docs.safing.io/)
```
Comment (dhaavi) [inline privacy/index.md:80]: Why is one linked, the other not? Is the a criteria?

File: privacy/index.md [hunk 2]
```diff
@@ -4,150 +4,562 @@ title: Privacy Policy
...
+a computer network and aims primarily to provide confidentiality and
+data integrity between two communicating computer applications.
 
-#### Location
+At rest: Hot data (in use) is protected on our server, cold data
+(backups) are encrypted.
 
-We use your IP address to identify the country from which you are accessing our services. Only half of your IP address saved.
+**Article 4. How we use your PII & Data**
+-----------------------------------------
```
Comment (dhaavi) [inline privacy/index.md:193]: Is this valid? You are making it bold _and_ a header.
I would suggest always using `##` in the correct level, as they are easier to edit and read. Especially if `#`s are used elsewhere.

File: terms/index.md [hunk 3]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
 
-- Forum: [https://discourse.safing.community/tos](https://discourse.safing.community/tos)
-- Portmaster: *coming soon*
-- Gate17: *coming soon*
-- This Website: *see below*
+Safing ICS Technologies GmbH welcomes your feedback, please send all
+feedback to: <feedback@safing.io>
 
-## Terms for this Website
+---------------------------------------
```
Comment (dhaavi) [inline terms/index.md:35]: If this is a `<hr>` line, three dashes should be sufficient: `---`.

File: terms/index.md [hunk 4]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
+
+-   You may not attempt to gain unauthorised access to any Service,
+    other accounts, computer systems or networks connected to our
+    network through hacking, password mining or any other means.
+
+-   You may not obtain or attempt to obtain any materials or information
+    through any means not intentionally made available through our
+    Services.
+
+<!-- -->
```
Comment (dhaavi) [inline terms/index.md:474]: ?

File: terms/index.md [hunk 5]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
+At present our forum is on the reddit platform, this can be reached by
+using the following link:
+
+<https://reddit.com/r/safing>
+
+we have no control on how the Reddit service collects, stores, shares,
+handles, or uses data, and you are advised to read any and all privacy
+policies of the sites you intend to use.
+
+**Article 16.** **Intellectual Property**
```
Comment (dhaavi) [inline terms/index.md:626]: Double Bold.

File: terms/index.md [hunk 6]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
+-----------------------------------------
+
+The trademark, SPN, logos, graphics and texts featured on any and all of
+our services are the intellectual property of Safing ICS Technologies
+GmbH and may not be reproduced, used or represented without the explicit
+permission, in writing via email to <office@safing.io> under threat of
+legal action, please read **[Article 13.](#terms-of-service) AUP**.
```
Comment (dhaavi) [inline terms/index.md:633]: This is not the correct answer for the AUP section (it has none?).

File: terms/index.md [hunk 7]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
+subscription.
+
+**Article 23. Warranties and Disclaimers**
+------------------------------------------
+
+**General Reference**. Safing ICS Technologies GmbH, cannot and will not
+be held responsible for the actions of its subscribed members whilst
+using any of our services including and not limited to the SPN, website
+and forums, such actions that are in breach of **[Article
+13.](#article-13-aup-acceptable-use-policy) AUP**, will result in a
```
Comment (dhaavi) [inline terms/index.md:743]: This looks better.

File: terms/index.md [hunk 8]
```diff
@@ -4,43 +4,881 @@ title: Terms of Service
...
+responsible or held liable in any way shape or form for, any delay or
+failure to perform to the extent, due to unforeseen circumstances beyond
+our reasonable control, including, and without limitations, failures of
+your telecommunications or internet service providers, Force Majeure,
+earthquakes, fires, floods, embargoes, labour disputes and strikes,
+riots, war, novelty of product manufacture, or any other unanticipated
+product development problems, and acts of civil and military
+authorities.
+
+**Reference** [**Article
```
Comment (dhaavi) [inline terms/index.md:760]: Boldness in References is handled very differently at different places.

File: privacy/index.md (hunk 1 above)
Comment (davegson) [reply privacy/index.md:80]: just an fyi, both generate a link ;)

**Pull request merged**